### DISCUSSION TITLE
How to stop the re-queuer process ?

### DISCUSSION THREAD

**Discussion by Orminor77** [postalserver/postal#1621] [Help with using Postal] [answered]
Hello,

I would like to know how to make sure that emails that have "soft fail" are not returned to the queue because it causes a blacklisting of my ip.

Thanks you in advance for your help !

**Comment by Orminor77** [comment]
Hey back,

I'm sorry to fast re-up my issue but can someone try to help me please, If it's not clear also tell me, I respond fast

Thanks you all

**Comment by willpower232** [comment, marked as answer]
the re queuer has several jobs so wouldn't recommend removing it, you could try lowering this value in your config to less than 18 but if you're being added to lists then its likely you have another problem

https://github.com/postalserver/postal/blob/6004767129de8c4f5521cc98ad0675c3473dbaf6/config/postal.defaults.yml#L14

**Reply by Orminor77** [reply]
Hello, thanks for your answer I appreciate !
It's what I search so the last question, is it possible to put maximum_delivery_attempts to 0 for no retry because FR ISP don't really like when we retry many times.

**Reply by willpower232** [reply]
You could try that but I'm not sure whether it will help your problem.

**Reply by Orminor77** [reply]
Ok thanks you I will try soon 👍
